tests/models/gptj/test_modeling_gptj.py

```diff
@@ -562,7 +562,7 @@ def test_gptj_sample(self):
         output_seq_strs = tokenizer.batch_decode(output_seq, skip_special_tokens=True)
         output_seq_tt_strs = tokenizer.batch_decode(output_seq_tt, skip_special_tokens=True)
 
-        if torch_device == "cuda":
+        if torch_device != "cpu":
```

Comment:
not really. Adding a comment:

```
# currently this expect value is only for `cuda`
```
would be nice

Reply:
Added necessary comment, thanks for your suggestions ❤️